darwin.m

```diff
@@ -0,0 +1,67 @@
+// +build darwin
+
+#import <Foundation/NSArray.h>
```

Comment:
I dislike the idea of embed another helper tool. But you inspired me to move the code to pac-cmd, and let it call itself with privileges, it works. Thanks @oxtoacart